base/sparse/sparsematrix.jl

```diff
@@ -335,7 +335,222 @@ function sparse_IJ_sorted!{Ti<:Integer}(I::AbstractVector{Ti}, J::AbstractVector
     return SparseMatrixCSC(m, n, colptr, I, V)
 end
 
-## sparse() can take its inputs in unsorted order (the parent method is now in csparse.jl)
+"""
+    sparse(I, J, V, [m, n, combine])
+
+Create a sparse matrix `S` of dimensions `m` x `n` such that `S[I[k], J[k]] = V[k]`. The
+`combine` function is used to combine duplicates. If `m` and `n` are not specified, they
+are set to `maximum(I)` and `maximum(J)` respectively. If the `combine` function is not
+supplied, duplicates are added by default. All elements of `I` must satisfy
+`1 <= I[k] <= m`, and all elements of `J` must satisfy `1 <= J[k] <= n`.
```

Comment:
> there have been updates to this docstring that you need to incorporate.
> 
> make sure the signature is consistent between here and the rst docs and run make docs to make sure you aren't undoing recent changes

First time playing in the `doc` sandbox, so apologies in advance for inevitable mistakes. I updated the signature for `sparse` in doc/stdlib/arrays.rst and issued `make julia-genstdlib`. No complaints about `sparse`, though complaints about other things:
```shell
UndefVarError(:build_sysimg)
WARNING: Mod Base build_sysimg
INFO: devdocs/sysimg.rst: no docs for build_sysimg(sysimg_path=default_sysimg_path, cpu_target="native", userimg_path=nothing; force=false) in Base

WARNING: Exported method missing doc for Base.StackTraces.StackTrace
WARNING: Exported method missing doc for StackFrame
WARNING: Exported method missing doc for Base.LibGit2.with
WARNING: Exported method missing doc for Base.Docs.doc
WARNING: Exported method missing doc for @threadcall

WARNING: Missing 5 exported doc strings
INFO: Missing 121 unexported doc strings
```
Guessing that constitutes success, there being no complaints about `sparse`? Subsequently issued `make docs`, and then `make -C doc html` and `make -C doc latex` for good measure; all seemed to complete fine. Should that do the trick? Will push after clarifying the `sparse!` docs. Thanks again!